doc/source/tutorials/spanning_trees/assets/spanning_trees.py

```diff
@@ -0,0 +1,38 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+import numpy as np
+
+g = ig.Graph.Lattice([6, 6], circular=False)
+
+# Optional: Rearrange the vertex ids to get a more interesting spanning tree
+layout = g.layout("grid")
+
+np.random.seed(0)
+permutation = np.random.permutation(36)
```

Comment:
In the vein of attempting to keep `numpy` out of the examples (not because I don't like `numpy` but because you need to know `numpy` as well to understand what's going on here):

```
from random import shuffle
[...]
permutation = list(range(g.vcount()))
shuffle(permutation)
```

This has the added bonus that you don't need to call `.tolist()` on the permutation.

Reply:
Done.